packages/ia-components/sandbox/language-select/language-select.less

```diff
@@ -1,5 +1,5 @@
 @dropdown_background_color: #222222;
-@text_color: #848484;
+@text_color: #b9b9b9;
```

Comment:
I've put in a request to have this color change approved by our design team before approving. Will let you know with a follow-up comment if this change is acceptable.

Reply:
Ok, from the design team: "We've been moving to #999999 for these. It's AA compliant for small text against the #222 background, and it's easier to differentiate from the #fff active text color." Updating this to #999999 and setting the image alt attribute to "" will make this ready to merge.